akka-d3-readside-cassandra/src/main/scala/akka/contrib/d3/CassandraReadSideProvider.scala

```diff
@@ -82,13 +75,13 @@ object CassandraReadSide {
           statements ← handler
             .asInstanceOf[EventStreamElement[Event] ⇒ Future[immutable.Seq[BoundStatement]]]
             .apply(element)
-        } yield statements :+ offsetDao.bindSaveOffset(element.offset)
+        } yield statements
       }
 
       def invokeHandler(handler: EventHandler[Event], elem: EventStreamElement[Event]): Future[Done] = {
         for {
           statements ← invoke(handler, elem)
-          done ← statements.size match {
+          _ ← statements.size match {
```

Comment:
You change the behaviour here, as storing the offset will not be batched with the other (possible) statements.